pkgs/top-level/all-packages.nix

```diff
@@ -17278,6 +17278,8 @@ with pkgs;
     plugins = [];
   };
 
+  pidgin-discord = callPackage ../applications/networking/instant-messengers/pidgin-plugins/pidgin-discord { };
```

Comment:
Why are you naming it `pidgin-discord` when the repo is called `purple-discord`?